prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -109,6 +109,25 @@ class FakeDataService : Logging {
             return "${context.patientName.username()}@email.com"
         }
 
+        // ID values typically come from valuesets, so we pass in the element
```

Comment:
Change this to kdoc.  https://kotlinlang.org/docs/kotlin-doc.html  
Makes sure to add the element argument and the return.